marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/step/getStepsByType.sjs

```diff
@@ -24,7 +24,7 @@ const Artifacts = require('/data-hub/5/artifacts/core.sjs');
 var propertiesToReturn;
 
 const response = {};
-const stepTypes = ['ingestion', 'mapping'];
+const stepTypes = ['ingestion', 'mapping', 'matching', 'merging', 'mastering'];
```

Comment:
Should 'mastering' step be present as well ? The users cannot create / edit one in HC but they can add it to flow ?